docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: model
+title: Sentence Entity Resolver for ICD10-CM (general 3 character codes)
+author: John Snow Labs
+name: sbiobertresolve_icd10cm_generalised_augmented
+date: 2023-05-24
+tags: [licensed, en, clinical, entity_resolution]
+task: Entity Resolution
+language: en
+edition: Healthcare NLP 4.4.2
+spark_version: 3.0
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps extracted medical entities to ICD10-CM codes using sbiobert_base_cased_mli Sentence Bert Embeddings. It predicts ICD codes up to 3 characters (according to ICD10 code structure the first three characters represent general type of the injury or disease).
+
+## Predicted Entities
+
+
+
+{:.btn-box}
+[Live Demo](https://demo.johnsnowlabs.com/healthcare/ER_ICD10_CM/){:.button.button-orange}
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/24.Improved_Entity_Resolvers_in_SparkNLP_with_sBert.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_icd10cm_generalised_augmented_en_4.4.2_3.0_1684930238103.zip){:.button.button-orange.button-orange-trans.arr.button-icon.hidden}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_icd10cm_generalised_augmented_en_4.4.2_3.0_1684930238103.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+sbiobertresolve_icd10cm_generalised resolver model must be used with sbiobert_base_cased_mli as embeddings ner_clinical as NER model. PROBLEM set in .setWhiteList().
```

Comment:
Please change here as:
`sbiobertresolve_icd10cm_generalised` resolver model must be used with `sbiobert_base_cased_mli` as embeddings. 

sbiobertresolve_icd10cm_generalised resolver model must be used with sbiobert_base_cased_mli as embeddings ~ner_clinical as NER model. PROBLEM set in .setWhiteList().~

We shouldn't restrict the NER model, any NER model can be used and labels whitelisted.

Reply:
Done